### PR TITLE
[unifi-poller] update to common@3.2.0 and unifi-poller@2.1.3

### PR DESCRIPTION
**Pull request by zamnuts**
Signed-off-by: Andrew Zammit <zammit.andrew@gmail.com>

**Description of the change**

- Upgrades `unifi-poller` dependency: k8s-at-home common library from version 2.5.0 to version 3.2.0 - BREAKING CHANGE
- Changes the default influxdb database name to match unifi-poller's default configuration
- Updates default unifi-poller version from 2.0.1 to 2.1.3

**Benefits**

Supports `custom` persistent types for ConfigMaps and Secrets, e.g:
```
persistence:
  unifi-poller-config:
    enabled: true
    type: custom
    readOnly: true
    volumeSpec:
      configMap:
        name: unifi-poller-config
    subPath:
      - path: unifi-poller.conf
        mountPath: /etc/unifi-poller/up.conf
      - path: uck-tls.pem
        mountPath: /etc/unifi-poller/uck-tls.pem
  unifi-poller-secrets:
    enabled: true
    type: custom
    readOnly: true
    volumeSpec:
      secret:
        secretName: unifi-poller-secrets
    subPath:
      - path: password.txt
        mountPath: /etc/unifi-poller/password.txt
```

**Possible drawbacks**

Breaking common schema changes going from 2.x to 3.x, hence the major version bump to 8.0.0, e.g. service ports, and upgrade strategy.

**Applicable issues**

none

**Additional information**

Tested install:
```
# helm ls -n unifi-poller -oyaml                                                           
- app_version: 2.1.3
  chart: unifi-poller-8.0.0
  name: unifi-poller
  namespace: unifi-poller
  revision: "5"
  status: deployed
  updated: 2021-06-20 20:53:10.836736822 -0700 MST
```

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [X] Variables are documented in the README.md (this can be done with using our helm-docs wrapper `./hack/gen-helm-docs.sh stable <chart>`)

### CODE REVIEW

**Review by bjw-s — CHANGES_REQUESTED** [review]
Hi! 👋 Thanks a lot for taking the time to write this PR! The changes look good overall I think. Could you please also update the README_CHANGELOG file and add the helm-docs strings to values.yaml? This is part of an effort we are making to improve our documentation 🙂

**Comment by zamnuts** [comment]
you run a tight ship. i updated the docs.

also, since this is a breaking change anyway, i opted to align the default influxdb database name to what unifi-poller expects out-of-the-box.

File: charts/stable/unifi-poller/values.yaml [hunk 1]
```diff
@@ -6,15 +6,18 @@
...
+  # -- Image [k8s pull policy](https://kubernetes.io/docs/concepts/containers/images/#updating-images).
   pullPolicy: IfNotPresent
-  tag: 2.0.1
+  # -- Image tag to deploy.
+  tag: 2.1.3
 
-strategy:
-  type: Recreate
+# -- [Deployment strategy](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy) used to replace old Pods by new ones.
+strategy: Recreate
```
Comment (bjw-s) [inline charts/stable/unifi-poller/values.yaml:17]:
```suggestion
```

This option was changed in the common v3 library. Sane defaults are now properly determined in there, so we can safely remove it from the subchart values.

File: charts/stable/unifi-poller/README_CHANGELOG.md.gotmpl [hunk 2]
```diff
@@ -9,19 +9,21 @@ All notable changes to this application Helm chart will be documented in this fi
...
+### [8.0.0]
 
 #### Added
 
 - N/A
 
 #### Changed
 
-- N/A
+- Upgrades unifi-poller dependency: k8s-at-home common library from version 2.5.0 to version 3.2.0 - BREAKING CHANGE
```
Comment (bjw-s) [inline charts/stable/unifi-poller/README_CHANGELOG.md.gotmpl:20]:
```suggestion
- **BREAKING**: Upgraded the common library dependency to version 3.2.0. This introduces several breaking changes (`service`, `ingress` and `persistence` keys have been refactored).
   Be sure to check out the [library chart](https://github.com/k8s-at-home/library-charts/blob/common-3.2.0/charts/stable/common/) for the up-to-date values.
```

We're trying to keep this one the same across all the charts

**Review by bjw-s — CHANGES_REQUESTED** [review]
This looks good! Thanks again for the PR! Two small changes requested, will merge once those are in 😄

**Review by bjw-s — APPROVED** [review]

**Comment by bjw-s** [comment]
@all-contributors please add @zamnuts for code

**Pull request merged**